Simplified.xcodeproj/project.pbxproj

```diff
@@ -2811,7 +2815,8 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = 5;
+				CODE_SIGN_STYLE = Manual;
```

Comment:
I think this change should be reverted, we shouldn't change the code signing

Reply:
should I revert the `CURRENT_PROJECT_VERSION` on this too?

Reply:
sure